### ISSUE TITLE
hot patch on 

### ISSUE THREAD

**Issue by StevenCTimm** [HEPCloud/decisionengine_modules#476]

**Comment by StevenCTimm** [comment]
We noticed that ever since we had upgraded to DE 1.7.5 and glideinwms 3.10.5 (on Sep 29 and Oct 2 on hepcsvc03 and cmsde01 respectively) the idtokens files had failed to be written in /var/lib/gwms-frontend/tokens.d
With the help of Namratha Urs we learned that a patch had been made to glideinwms 3.10.5 such that it is now
writing string objects instead of bytes.  The glide_frontend_element.py in glideinwms frontend had been patched,
as had the 2.0 DE trunk, but the patch had not been back ported to 1.7.5.

The change is the following:


Line 1343 of glide_frontend_element.py originally read

with tempfile.NamedTemporaryFile(mode="wb", delete=False, dir=tkn_dir) as fd:

We changed it to say

with tempfile.NamedTemporaryFile(mode="w", delete=False, dir=tkn_dir) as fd:

Note the change of mode

We are running with patched 1.7.5 now on all production decision engines.
